Tribler/Core/Libtorrent/LibtorrentMgr.py

```diff
@@ -301,6 +301,8 @@ def remove_torrent(self, torrentdl, removecontent=False):
                 self.torrents[infohash][1].remove_torrent(handle, int(removecontent))
                 out = self.torrents[infohash][0].deferred_removed
                 self._logger.debug("remove torrent %s", infohash)
+                # del self.torrents[infohash]; Don't remove the torrent here yet, let it be deleted
```

Comment:
The code in this line comment confused me. Please remove it here and add it as general info to the docstring of the function.
Something like:

```python
def remove_torrent(self, torrentdl, removecontent=False):
"""
Start removing a torrent, the process is completed when a 'torrent_removed_alert' is received in 'process_alert'.
"""
```